apps/web/src/server/applications/case/examination-timetable/applications-timetable.controller.js

```diff
@@ -218,10 +232,13 @@ export async function unpublishApplicationsCaseTimetables(_, response) {
 /**
  * View the delete page for the examination timetable for a single case
  *
- * @type {import('@pins/express').RenderHandler<{timetableItem: Record<string, string>}, {}, {}, {}, {timetableId: string}>}
+ * @type {import('@pins/express').RenderHandler<{timetableItem: Record<string, string>, isCaseWelsh: boolean}, {}, {}, {}, {timetableId: string}>}
  */
 export async function viewApplicationsCaseTimetableDelete(request, response) {
 	const timetableItem = await getCaseTimetableItemById(+request.params.timetableId);
+	const isCaseWelsh = await isCaseRegionWales(
+		response.locals.case?.geographicalInformation?.regions
+	);
```

Comment:
Now this won't need an `await`
```suggestion
	const isCaseWelsh = isCaseRegionWales(
		response.locals.case?.geographicalInformation?.regions
	);
```